components/collator/src/options.rs

```diff
@@ -282,7 +300,7 @@ pub enum BackwardSecondLevel {
 /// ## Case Level
 ///
 /// See the [spec](https://www.unicode.org/reports/tr35/tr35-collation.html#Case_Parameters).
-/// This is the BCP47 key `kc`. The default is `false` (off).
+/// This is the BCP47 key `kc`. The default is `CaseLevel::Off`.
```

Comment:
nit: use [\`\`] for automatic cross references everywhere

Reply:
Done.